test/openssl/utils.rb

```diff
@@ -119,6 +119,18 @@ def libressl?(major = nil, minor = nil, fix = nil)
     return false unless version
     !major || (version.map(&:to_i) <=> [major, minor, fix]) >= 0
   end
+
+  def omit_on_fips
+    # The password based encryption used in the PEM format uses MD5 for
+    # deriving the encryption key from the password, and MD5 is not
+    # FIPS-approved.
+    #
+    # See https://github.com/openssl/openssl/discussions/21830#discussioncomment-6865636
+    # for details.
+    if OpenSSL.fips_mode
+      omit "The encryption used in the test is not approved in FIPS"
+    end
+  end
```

Comment:
Please move this to OpenSSL::TestCase because this method doesn't make sense as a class method. (OpenSSL::TestCase will include and extend OpenSSL::TestUtils.)

Reply:
Ah all right. I will move the method into the `OpenSSL::TestCase`.

Reply:
I moved the `omit_on_fips` method into the `OpenSSL::TestCase`.